src/desktop/config.coffee

```diff
@@ -105,6 +105,7 @@ module.exports =
   STRIPE_PUBLISHABLE_KEY: null
   TEAM_BLOGS: '^\/life-at-artsy$|^\/artsy-education$|^\/gallery-insights$|^\/buying-with-artsy$'
   TARGET_CAMPAIGN_URL: '/seattle-art-fair-2017'
+  TRACK_PAGELOAD_PATHS: null
```

Comment:
Does the default here make sense to be `[]` instead? Since you have a `_.each` above. Otherwise that might be a crash which would require you to actually set this pre-deploy, etc...

Reply:
There's a check above to make sure this variable exists (and returns if not) so I think this is okay.